README.md

```diff
@@ -20,6 +20,20 @@ Use `--dry-run` to check result without modifying data.
 ## Force
 Use `--force` to skip the confirmation prompt before modifying data.
 
+## Additional options for `eav:attributes:restore-use-default-value`
+
+### Always remove
+Use `--always_restore` to remove all values, even if the scoped value is not equal to the base value.
+
+### Store codes
+Use `--store_codes=your_store_code` to only remove values for this store.
+
+### Include attributes
+Use `--include_attributes=some_attribute,some_other_attribute` to only delete values for these attributes.
```

Comment:
if you give in a number (or any other non-existing store/attributecode )  it will throw an exception
```
bin/magento eav:attributes:restore-use-default-value --store_codes=123

                                                                               
  The store that was requested wasn't found. Verify the store and try again.   
   | store ID: 123                                                             
```
That said, this should state code, not ID, will push a fix